html-block-and-inline/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+  <title>Document</title>
+</head>
+<body>
+  <h1>Blue Americana Snuggie&reg; Blanket</h1>
+  <h2>by Snuggie Store</h2>
+  <p>Price: <s>$19.99</s>   $14.99</p>
+  <p>Part of our new Americana collection, this <i>beautiful</i> blue is already a customer favorite!</p>
+  <p>Enjoy complete <b>Freedom of Movement!</b></p>
```

Comment:
Same here, it is best to use `<strong>` over `<b>` for the same reason mentioned above.